subprojects/base-services/src/main/java/org/gradle/api/internal/DocumentationRegistry.java

```diff
@@ -44,6 +44,11 @@ public String getDslRefForProperty(Class<?> clazz, String property) {
         return String.format("https://docs.gradle.org/%s/dsl/%s.html#%s:%s", gradleVersion.getVersion(), className, className, property);
     }
 
+    public String getDslRef(Class<?> clazz, String anchor) {
+        String className = clazz.getName();
+        return String.format("https://docs.gradle.org/%s/dsl/%s.html#%s", gradleVersion.getVersion(), className, anchor);
```

Comment:
```suggestion
        return String.format("https://docs.gradle.org/%s/javadoc/%s.html#%s", gradleVersion.getVersion(), className, anchor);
```

I don't think there is a `CopySpec` DSL entry, only a Javadoc entry. This is what you are looking for, right: https://docs.gradle.org/current/javadoc/org/gradle/api/file/CopySpec.html#getDuplicatesStrategy-- ?

Reply:
Darn I need my glasses ... Fixed by using the DSL doc for `Copy`.